cinema/urls.py

```diff
@@ -15,6 +17,8 @@
 router.register("cinema_halls", CinemaHallViewSet)
 router.register("movies", MovieViewSet)
 router.register("movie_sessions", MovieSessionViewSet)
+router.register("orders", OrderViewSet)
+router.register("tickets", TicketViewSet)
 
 urlpatterns = [path("", include(router.urls))]
```

Comment:
No need to include when base path is not provided. Use:
urlpatterns = router.urls